src/figuresClasses.ts

```diff
@@ -1,19 +1,111 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
 
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
+
+  public color: Color;
+
+  public a: number;
+
+  public b: number;
+
+  public c: number;
+
+  constructor(
+    color: Color,
+    a: number,
+    b: number,
+    c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Invalid side(`s) length');
+    }
+
+    const logestSide = Math.max(a, b, c);
+
+    if (logestSide >= a + b + c - logestSide) {
+      throw new Error('Invalide sides values');
+    }
+
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
```

Comment:
![image](https://github.com/mate-academy/ts_figures_classes/assets/54860641/9707da3f-6d57-473c-80bc-bccd1dcf9e91)
Fix this in all code.